### PR TITLE
Clinic database

### PR DESCRIPTION
**Pull request by iankiema**
The following files have been added and updated 
-Schema based on diagram.sql
-Readme file.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+CREATE TABLE invoice_items (
+	id INTEGER PRIMARY KEY,
+	unit_price DECIMAL,
+	quantity INTEGER,
+	total_price DECIMAL,
+	invoice_id INTEGER,
+	treatment_id INTEGER,
+	FOREIGN KEY (invoice_id) REFERENCES invoices (id),
+	FOREIGN KEY (treatment_id) REFERENCES treatments (id)
+);
```
Comment (ridwanediallo) [inline schema_based_on_diagram.sql]: - [ ] Great job creating the tables :clap: . However, there are two missing requirements that you'd need to implement in order to get this milestone complete.
 1. As stated in the requirements, please add indexes for all the foreign keys. :ok_hand: 😊
    #### requirement :point_down: 
Screenshot from 2023-10-03 14-42-52
2. In the requirements, it is stated that we should create join tables from many-to-many relationships. And we have a `many-to-many` relationship between the `medical_histories` table and the `treatments` table. So I would kindly suggest you create the join table.  :ok_hand: 😊

   #### requirement :point_down: 
Screenshot from 2023-10-03 14-37-26

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi :wave:  team, 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- The tables have been created based on the diagram. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+CREATE INDEX idx_invoice_id_invoice_items ON invoice_items(invoice_id);
+CREATE INDEX idx_treatment_id_invoice_items ON invoice_items(treatment_id);
+
+CREATE TABLE medical_history_treatments (
+    medical_history_id INTEGER,
+    treatment_id INTEGER,
+    PRIMARY KEY (medical_history_id, treatment_id),
+    FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id),
+    FOREIGN KEY (treatment_id) REFERENCES treatments(id)
+);
```
Comment (Alusp) [inline schema_based_on_diagram.sql]: - [ ] Good job in implementing the changes requested by the previous reviewer, 👏 , however, as he mentioned earlier, and as part of the project requirement, Please ensure that you create indexes for the Foreign keys in your Join table. 🙂

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi  @ngalamac and @iankiema, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
close

 ## Highlights
✔️ No linter Errors
✔️ Descriptive README
✔️ Descriptive PR title and description
✔️ Completed most of the Tasks

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @iankiema & @ngalamac 

Good job implemented the suggested changes💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Created tables according to diagram
-  ✅ Great work using the right GitFlow 
-  ✅ Implemented suggested changes

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**